core/src/main/java/org/elasticsearch/search/suggest/SuggestUtils.java

```diff
@@ -171,12 +173,12 @@ public static SuggestMode resolveSuggestMode(String suggestMode) {
         }
     }
 
-    public static Suggest.Suggestion.Sort resolveSort(String sortVal) {
+    public static TermSuggestionBuilder.SortBy resolveSort(String sortVal) {
```

Comment:
These helpers are in the enums themselves as the `resolve` static methods.  I realized this method can go away and I will remove it because we are now just using this one sort enum to represent everything.  However, the same does not apply for the StringDistance and SuggestMode helper methods, because these helper methods convert the string to the direct Lucene representation, not the enum.  These methods will eventually go away when we remove all parser -> Lucene representation code in favor of going through the builders.